gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Mon Nov 06 17:09:51 GMT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.3-bin.zip
```

Comment:
Did you consciously make these changes, or is your local gradle wrapper once downloaded different?

Reply:
sorry about that.  I thought I'd excluded the gradle wrapper folder entirely.  Accident on my side if so.

Side note (and why I missed it):  I always include it in my .ignore file since i view keeping the build tool in the repo as ... weird, rather I just have a gradle wrapper command to run when first pulling.  Of course that means I've installed gradle, but then, most envs require you install the build tool.  If some user is developing java they've already downloaded the JDK - and not just the JRE - and so they're 90% there.